docs/src/main/asciidoc/datasource.adoc

```diff
@@ -515,8 +513,6 @@ public class MyConsumer {
 }
 ----
 
-Alternatively, you may define a xref:cdi.adoc#ok-you-said-that-there-are-several-kinds-of-beans[CDI bean producer] for the default datasource redirecting to the currently active named datasource, so that it can be injected directly, like this:
-
```

Comment:
Was this removed for a reason?

Reply:
Good point. I have it fixed but forgot to apply it. Returning revamped. THX!